lib/std/Io/Reader.zig

```diff
@@ -16,9 +16,9 @@ pub const Limited = @import("Reader/Limited.zig");
 vtable: *const VTable,
 buffer: []u8,
 /// Number of bytes which have been consumed from `buffer`.
-seek: usize,
+seek: usize = 0,
```

Comment:
https://ziglang.org/documentation/master/#Faulty-Default-Field-Values

Reply:
I see, because ``seek <= end``. Okay!

Pushed two new commits, see if that resolves your issue with the PR!